### ISSUE TITLE
Confusion in the display of the loss while training sphereface-plus

### ISSUE THREAD

**Issue by Maharshi-Aupera** [wy1iu/sphereface-plus#6]
Thanks for your great work. I was just wondering about the following snippet:
```
I0204 14:26:03.838340  6323 solver.cpp:218] Iteration 10980 (0.968637 iter/s, 20.6476s/20 iters), loss = 14.7497
I0204 14:26:03.838383  6323 solver.cpp:237]     Train net output #0: lambda = 5
I0204 14:26:03.838428  6323 solver.cpp:237]     Train net output #1: maximize_inter_class_dist = 14.3479 (* 1 = 14.3479 loss)
I0204 14:26:03.838443  6323 solver.cpp:237]     Train net output #2: softmax_loss = 0.401827 (* 1 = 0.401827 loss)
I0204 14:26:03.838527  6323 sgd_solver.cpp:105] Iteration 10980, lr = 0.01
I0204 14:26:22.732009  6323 solver.cpp:447] Snapshotting to binary proto file result/m_single_model_iter_11000.caffemodel
I0204 14:26:23.138741  6323 sgd_solver.cpp:273] Snapshotting solver state to binary proto file result/m_single_model_iter_11000.solverstate
I0204 14:26:24.393401  6323 solver.cpp:310] Iteration 11000, loss = 13.0881
I0204 14:26:24.393429  6323 solver.cpp:315] Optimization Done.
I0204 14:26:24.788636  6323 caffe.cpp:259] Optimization Done.
```

There is this `Train net output #2: softmax_loss = 0.401827 (* 1 = 0.401827 loss)` and at the end it also shows `I0204 14:26:24.393401  6323 solver.cpp:310] Iteration 11000, loss = 13.0881`. What is this last loss `13.0881`? - I think that this loss is a combination of a few other losses. Should I take care of this loss or the `softmax_loss` during the training.

**Comment by liulixinkerry** [comment]
I think you may ignore this one, it just the sum of MHE loss and softmax
loss. Just pay more attention to softmax loss.


Maharshi-Aupera <notifications@github.com> 于 2020年2月5日周三 07:29写道：

**Comment by Maharshi-Aupera** [comment]
Thanks for the prompt response.
